scripts/install/lazylibrarian.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+# LazyLibrarian install script for swizzin
+# Author: Aethaeran
+
+##########################################################################
+# References
+##########################################################################
+
+# https://github.com/Aethaeran/swizzin/blob/feat/jdownloader/scripts/install/jdownloader.sh
+# https://lazylibrarian.gitlab.io/install/
+# https://www.reddit.com/r/LazyLibrarian/comments/jw5e22/lazylibrarian_calibre_calibreweb_booksonic/
+# https://github.com/lazylibrarian/LazyLibrarian
+# https://gitlab.com/LazyLibrarian/LazyLibrarian
+# https://lazylibrarian.gitlab.io/config_commandline/
+# https://github.com/swizzin/swizzin/pull/743
+# https://lazylibrarian.gitlab.io/config_users/
+# https://lazylibrarian.gitlab.io/config_downloaders/
+# https://www.reddit.com/r/LazyLibrarian/
+# https://lazylibrarian.gitlab.io/api/
+
+##########################################################################
+# Import Sources
+##########################################################################
+
+. /etc/swizzin/sources/functions/color_echo
+. /etc/swizzin/sources/functions/pyenv
+. /etc/swizzin/sources/functions/users
+. /etc/swizzin/sources/functions/utils
```

Comment:
These are missing the `#shellcheck source=` directives, e.g.

```bash
    #shellcheck source=sources/functions/mono
    . /etc/swizzin/sources/functions/mono
```

Reply:
Most of them are imported by box already anyways. I just put them there so I would remember to source them to CLI when testing individual blocks.

Will add the "shellcheck sources" to the ones that remain after verifying which are necessary or not.

Reply:
Done.
PyCharm's shellcheck never says anything about "shellcheck source". So I never would have caught this on my own. Thanks a bunch.